marbles/scripts/index.ts

```diff
@@ -0,0 +1,44 @@
+import { renderMarbleDiagram } from 'swirly-renderer-node';
+import { readdir, readFileSync, writeFileSync } from 'fs';
+import { join } from 'path';
+import { parseMarbleDiagramSpec } from 'swirly-parser';
+import { styles as defaultLight } from 'swirly-theme-default-light';
```

Comment:
It's fine if you don't provide all styles. Swirly will extend the default light theme with whatever you do provide.

Reply:
oh that's good to know. Thanks